test/e2e/tcp/connectivity.go

```diff
@@ -174,12 +174,10 @@ func createPods(p *ConnectivityTestParams) (*framework.NetworkPod, *framework.Ne
 		framework.Logf("Will send traffic from IP: %v", sourceIP)
 	}
 
-	By(fmt.Sprintf("Waiting for the connector pod %q on node %q to exit, returning what connector sent",
-		connectorPod.Pod.Name, connectorPod.Pod.Spec.NodeName))
+	By(fmt.Sprintf("Waiting for the connector pod %q to exit, returning what connector sent", connectorPod.Pod.Name))
```

Comment:
Why did you remove printing the node name here? Isn't that contrary to the purpose of this PR?

Reply:
Cos' it will be duplicated as its getting printed as part of connectorPod.AwaitFinish() api below. 

Please see the sample output I pasted in the PR:
https://github.com/submariner-io/shipyard/pull/980#issuecomment-1279352310

Reply:
Right but that's only if `verbose` is enabled, which it is for CI but I assume not for `subctl`. I don't think it hurts to duplicate it anyway - we print the pod name in  both places so it would be consistent.

Reply:
If we do not pass "--verbose" as part of `subctl verify ...` command, we do not get any info including these log messages right? 
See sample output: https://pastebin.com/raw/yPg8Sd4q

Reply:
OK right. Still IMO it doesn't hurt to print it anyway - as I said we duplicate printing the pod name so why not the node name as well since we did before.  If we want to be consistent then we shouldn't duplicate the pod name either. Anyway if you feel strongly about it then I'm fine either way.

Reply:
The main issue is that sometimes its empty at this stage and could be misleading. 

```
Waiting for the connector pod "tcp-check-podc69qn" on node "" to exit, returning what connector sent
INFO: Pod "tcp-check-podc69qn" output:
```

The latest pod info gets updated as part of connectorPod.AwaitFinish() API where we have the node details associated with the pod.